### ISSUE TITLE
Where is the code for DREN training

### ISSUE THREAD

**Issue by Cli98** [zhangjunyi1225054736/ACDT#1]
Hi and can I ask where is the code for DREN training? You mentioned ssd but I did not see any files in your folder.

**Comment by Cli98** [comment]
And if you use SSD modular of pytorch to train DREN, can you summarize what's the input parameters and output parameters for your DREN?

**Comment by Cli98** [comment]
@zhangjunyi1225054736 can you briefly answer my question?

**Comment by zhangjunyi1225054736** [comment]
> @zhangjunyi1225054736 can you briefly answer my question?

很抱歉，之前没有看到你的回复，这份代码还没有完全开源。ssd那个代码是直接用的开源的，github上有，我们就没有放出来了，参数论文里说了，大部分是按照默认的参数。

**Comment by Cli98** [comment]
Thank you for your kindly response.
